odoo/addons/base/tests/test_translate.py

```diff
@@ -659,9 +659,9 @@ def test_fields_view_get(self):
         info = model.fields_get(['name'])
         self.assertEqual(info['name']['string'], LABEL)
 
-        # check that fields_view_get() also returns the expected label
-        info = model.fields_view_get()['fields']
-        self.assertEqual(info['name']['string'], LABEL)
+        # check that get_views() also returns the expected label
+        info = model.get_views([(False, 'form')])
+        info['models'][model._name]['name']['string']
```

Comment:
Hey, you dropped the test!
```suggestion
        self.assertEqual(info['models'][model._name]['name']['string'], LABEL)
```